src/instructlab/model/evaluate.py

```diff
@@ -87,6 +87,19 @@ def get_evaluator(
                 fg="red",
             )
             raise click.exceptions.Exit(1)
+        if os.path.exists(model):
+            model = pathlib.Path(model)
+            valid_model = False
+            if model.is_dir():
+                valid_model = backends.is_model_safetensors(model)
+            elif model.is_file():
+                valid_model = backends.is_model_gguf(model)
+            if not valid_model:
+                click.secho(
+                    "MTBench and MTBenchBranch need to be passed either safetensors or a GGUF file",
```

Comment:
nit: for a more clueless user (like me), it could be of help to explicitly say here (and below) in the error message that "safetensors" is supposed to be a directory (and not e.g. an actual .safetensors file.)

Something like: "MTBench and MTBenchBranch need to be passed either a safetensors directory or a GGUF file"

Reply:
I like it! Will update with that text